app/modules/munkiinfo/scripts/munkiinfo.py

```diff
@@ -106,9 +128,13 @@ def munkiinfo_report():
     if AppleCatalogURL == 'None':
         AppleCatalogURL = ''
 
+    middleware_name, middleware_version = middleware_checks()
+
     report = {
         'AppleCatalogURL': AppleCatalogURL,
         'munkiprotocol': munkiprotocol,
+        'MiddlewareName': middleware_name,
+        'MiddlewareVersion': middleware_version,
     }
```

Comment:
As discussed on slack we're doing Name-Version, so something like middleware_s3-1.0 will show up. I beefed up the logic to return a blank string when the preference isn't read properly.